contracts/hub/virtual_balance/src/execute.rs

```diff
@@ -86,44 +106,70 @@ pub fn execute_burn(
 }
 
 pub fn execute_transfer(
-    deps: DepsMut,
+    deps: &mut DepsMut,
     info: MessageInfo,
     msg: ExecuteTransfer,
```

Comment:
I think we should rename this variable to avoid `msg.msg` situations. 
Maybe to `transfer_msg` and within ExecuteTransfer, rename msg to `forward_msg`